backend/crowdin/events.go

```diff
@@ -0,0 +1,106 @@
+package crowdin
+
+import (
+	"context"
+	"github.com/ansel1/merry/v2"
+	"github.com/bcc-code/brunstadtv/backend/common"
+	"github.com/bcc-code/brunstadtv/backend/directus"
+	"github.com/bcc-code/brunstadtv/backend/events"
+	"github.com/bcc-code/mediabank-bridge/log"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/go-resty/resty/v2"
+	"github.com/samber/lo"
+)
+
+var CollectionNotSupported = merry.Sentinel("collection not supported")
+
+type PubSubEvent struct {
+	Type string `json:"type"`
+}
+
+type services interface {
+	GetDirectusClient() *resty.Client
+}
+
+type config interface {
+	GetCrowdinToken() string
+	GetCrowdinProjectIDs() []int
+}
+
+func HandleEvent(ctx context.Context, services services, config config, event cloudevents.Event) (err error) {
+	client := New(config.GetCrowdinToken(), ClientConfig{
+		ProjectIDs: config.GetCrowdinProjectIDs(),
+	})
```

Comment:
Would it make sense for this to be global? Then you could also potentially use it as a lock to prevent multiple uses on the same instance at least.